### PR TITLE
[Australia] - Announce speakers and schedule

### PR DESCRIPTION
**Pull request by swapnilogale**
----
📚 Documentation preview 📚: https://writethedocs-www--2232.org.readthedocs.build/

### CODE REVIEW

File: docs/_data/australia-2024-config.yaml [hunk 1]
```diff
@@ -28,8 +28,8 @@ buttons:
   #     link: /news/announcing-virtual-conf-dates-tickets
   #   - text: Read our conference guide
   #     link: /welcome-wagon
-  #  - text: See the Schedule!
-  #    link: /schedule
```
Comment (plaindocs) [inline docs/_data/australia-2024-config.yaml:32]: You'll want to comment out one of the other ones when you do this, so only two display.

File: docs/conf/australia/2024/news/announcing-speakers-schedule.rst [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Our Australian conference is happening in just over 2 months, and we're excited to see everyone again in {{city}}.
+
+Full speaker line-up
+--------------------
+
+Each year we bring a wide range of voices to the Write the Docs. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
+
+We're really excited about the `presentations <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ we've got this year:
+
+.. datatemplate::
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/announcing-speakers-schedule.rst:22]: It's gonna make life a lot easier if you add the invited speakers to Pretalx, and we extract the data from there in the same way.

Comment (swapnilogale) [reply docs/conf/australia/2024/news/announcing-speakers-schedule.rst:22]: Yep, will do once I get their bios and talk abstracts

**Review by plaindocs — APPROVED** [review]
Looks ok to me

**Pull request merged**